docs/high-availability/kubeconfig

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Again just a minimal impl that I think we may want to expand on so that's why I created as a separate file